### ISSUE TITLE
Audit heroku postgres instances

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#5937]
## Description

Currently we have 4 databases on heroku for commonwealth in production, only one of which is the "production database". We also have 3 databases on commonwealth beta = QA, 4 on frick, and 2 on frack.

Output of this spike should be to determine the function of each of these databases, and recommendations for what action we should take on the database (keep or remove).

Timebox: 2 hours

**Comment by kurtisassad** [comment]
# commonwealthapp

 DATABASE/HEROKU_POSTGRESQL_BROWN/HEROKU_POSTGRESQL_PUCE - our main database

HEROKU_POSTGRESQL_GREEN - Seems to be an outdated duplicate database (not following any other dbs). (Most recent address 2023-03-28 15:30:34.459+00). 

DATADOG/HEROKU_POSTGRESQL_TEAL/HEROKU_POSTGRESQL_WHITE - Seems to be an outdated duplicate database. (Most recent address 2023-03-28 16:02:40.072+00). Our datadog seems to be using it. Note that our HEROKU_POSTGRESQL_RED (main demo db) is following this one.

HEROKU_POSTGRESQL_BLACK/READONLY - I am not sure what this DBs purpose is, it has a totally different schema, last row inserted on “2021-07-27 23:40:51.269954” with most common tables include: (topic_views, unsubscribe_keys, application_requests, post_timings, schema_migrations, schema_migration_details, stylesheet_cache, incoming_links.

# commonwealth-beta

DATABASE - this is the database that beta uses.

HEROKU_POSTGRESQL_RED - some database following the outdated commonwealthapp HEROKU_POSTGRESQL_TEAL database. Unsure of use.

HEROKU_POSTGRESQL_MAROON - This is following commonwealthapp DATABASE, our main db.

# commonwealth-frick
 
DATABASE/HEROKU_POSTGRESQL_IVORY/HEROKU_POSTGRESQL_MAUVE/HEROKU_POSTGRESQL_ROSE - the main database for frick

HEROKU_POSTGRESQL_BRONZE - An empty database, no connections or data.

HEROKU_POSTGRESQL_COPPER - no connections. Write limit reached for plan (max rows exceeded). Last address updated 2019-04-21 18:36:03.158+00)

HEROKU_POSTGRESQL_TEAL - no connections. Write limit reached for plan (max rows exceeded). Same last address updated 2019-04-21 18:36:03.158+00)

# commonwealth-frack

DATABASE/DATADOG/HEROKU_POSTGRESQL_TEAL - main database

HEROKU_POSTGRESQL_WHITE - Following outdated HEROKU_POSTGRESQL_TEAL database.


# TLDR removals:
We can confidently remove:
#### commonwealth-frick
HEROKU_POSTGRESQL_COPPER, HEROKU_POSTGRESQL_BRONZE, HEROKU_POSTGRESQL_TEAL (commonwealth-frick)

We can probably remove:
#### commonwealthapp
HEROKU_POSTGRESQL_GREEN (Need to figure out what its for), 

#### commonwealth-beta
HEROKU_POSTGRESQL_RED

#### commonwealth-frack
HEROKU_POSTGRESQL_WHITE

# TLDR refactor:
We should move DATADOG database to reading off of HEROKU_POSTGRES_MAROON and remove HEROKU_POSTGRESQL_TEAL since it is outdated use.

**Comment by jnaviask** [comment]
Re HEROKU_POSTGRESQL_BLACK -- this is a database containing a discourse migration @kurtisassad

**Comment by jnaviask** [comment]
I believe the databases from 2023-03-28 were created by me as backups when I had to do a hard db rollback following a bad production deploy. We should be able to remove them.

**Comment by jnaviask** [comment]
Current plan is to remove the following:
- GREEN/WHITE/BLACK on production
- RED on QA
- BRONZE/COPPER/TEAL on frick
- WHITE on frack

**Comment by jnaviask** [comment]
Databases have been deleted.
